ext/php5/logging.c

```diff
@@ -57,7 +57,7 @@ int ddtrace_bgs_logf(const char *fmt, ...) {
             // todo: we only need 20-ish for the main part, but how much for the timezone?
             // Wish PHP printed -hhmm or +hhmm instead of the name
             char timebuf[64];
-            int time_len = strftime(timebuf, sizeof timebuf, "%d-%m-%Y %H:%m:%S %Z", now_local);
+            int time_len = strftime(timebuf, sizeof timebuf, "%d-%b-%Y %H:%M:%S %Z", now_local);
```

Comment:
Very good point. This seems to be the default format of the fpm sapi (the most used). Under the hood this is the method used https://heap.space/xref/PHP-7.4/main/main.c?r=39ddf6b8#887 that uses `M` for the format with a `DateTime::format` convention https://www.php.net/manual/en/datetime.format.php.

Otherwise you would have BGS logs with months rendered differently than the main process.

Reply:
Love it! 💯